README.md

```diff
@@ -145,7 +145,7 @@ Have a bug or a feature request? Please first read the [issue guidelines](https:
 
 Bootstrap's documentation, included in this repo in the root directory, is built with [Hugo](https://gohugo.io/) and publicly hosted on GitHub Pages at <https://getbootstrap.com/>. The docs may also be run locally.
 
-Documentation search is powered by [Algolia's DocSearch](https://community.algolia.com/docsearch/). Working on our search? Be sure to set `debug: true` in `site/assets/js/search.js`.
+Documentation search is powered by [Algolia's DocSearch](https://www.algolia.com/doc/). Working on our search? Be sure to set `debug: true` in `site/assets/js/search.js`.
```

Comment:
IMO the URL is rather https://docsearch.algolia.com/ isn't it?

Reply:
Yeah, I guess so